pkgs/development/python-modules/skein/skeinjar.nix

```
@@ -1,21 +1,19 @@
-{ callPackage, stdenv, maven, src, version }:
+{ fetchurl, unzip, stdenv, version, jarHash }:
 
-let
-  skeinRepo = callPackage ./skeinrepo.nix { inherit src version; };
-in
 stdenv.mkDerivation rec {
   pname = "skein.jar";
+  inherit version;
 
-  inherit version src;
+  src = fetchurl {
+    url = "https://files.pythonhosted.org/packages/py2.py3/s/skein/skein-${version}-py2.py3-none-any.whl";
+    hash = jarHash;
+  };
 
-  nativeBuildInputs = [ maven ];
-
-  buildPhase = ''
-    mvn --offline -f java/pom.xml package -Dmaven.repo.local="${skeinRepo}" -Dskein.version=${version} -Dversion=${version}
-  '';
+  dontUnpack = true;
+  dontBuild = true;
 
   installPhase = ''
-    # Making sure skein.jar exists skips the maven build in setup.py
-    mv java/target/skein-*.jar $out
+    ${unzip}/bin/unzip ${src}
```

```suggestion
    unzip ${src}
```
Please add unzip to nativeBuildInputs.

---

Done. What's the significance of this change?

---

More a style thing. I don't think there is a difference.